models/r21d/extract_r21d.py

```diff
@@ -15,24 +15,33 @@
 PRE_CENTRAL_CROP_SIZE = (128, 171)
 KINETICS_MEAN = [0.43216, 0.394666, 0.37645]
 KINETICS_STD = [0.22803, 0.22145, 0.216989]
+R21D_MODEL_LIST = [
```

Comment:
A bit of a nitpick but we could do it as a nested `dict` (`{<model_name>: {<param>: <value>}`. This would simplify the code in `__init__`.

```python
R21D_MODEL_DICT = {
    'r2plus1d_34_32_kinetics': {
        'repo': ...,
        'step_size': ...,   
    }
}
```